static/src/javascripts/projects/common/modules/discussion/comments.js

```diff
@@ -413,7 +413,8 @@ Comments.prototype.replyToComment = function(e) {
 Comments.prototype.reportComment = function(e) {
     e.preventDefault();
 
-    var commentId = e.currentTarget.getAttribute('data-comment-id');
+    var self = this,
+        commentId = e.currentTarget.getAttribute('data-comment-id');
 
     $('.js-report-comment-form').first().each(function(form) {
         bean.one(form, 'submit', function(e) {
```

Comment:
kinda unrelated, but `bean.one` means that you'll be able submit only once. If there's an error, the second time I click on the submit button either nothing happens, or I'm redirected to the form location (hopefully that won't trigger other errors)